src/WidgetBase.ts

```diff
@@ -198,13 +191,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 		this._properties = <P> {};
```

Comment:
Actually `hasOwnProperty` is used by Maquette on properties so we need it to be an object.